### ISSUE TITLE
Model configuration of 2-D data

### ISSUE THREAD

**Issue by deweihu96** [AgatheSenellart/MultiVae#49]
Hi,

Thanks for making this awesome repository!

I'm testing my data based on the turtorial, and I have 1000 samples for two modalities; one has 5 features, and another one has 10 features. 

I got the errors when I configed in this way

```
model_config = MVAEConfig(
    n_modalities = 2,
    input_dims= {'mod1': (5), 
                           'mod2': (10)},
    latent_dim= 8,
)

model = MVAE(model_config)

# Define a trainer


training_config = BaseTrainerConfig(
    learning_rate=1e-3, 
    num_epochs=1
)

trainer = BaseTrainer(model , dataset, training_config=training_config)
```

error:
Exception: Error when calling forward method from model. Potential issues: 
 - Wrong model architecture -> check encoder, decoder and metric architecture if you provide yours 
 - The data input dimension provided is wrong -> when no encoder, decoder or metric provided, a network is built automatically but requires the shape of the flatten input data.
Exception raised: <class 'IndexError'> with message: The shape of the mask [64, 1] at index 1 does not match the shape of the indexed tensor [64, 700, 5] at index 1

Thanks for you time and help in advance!

**Comment by AgatheSenellart** [comment]
Hi, 
Thank you so much for reaching out to us and using our repository ! 

Here are the possible issues with your application:

- The input dimensions must be tuples so instead of (5) for the first modality, it should be (5,)
- Do you use an incomplete dataset with masks ? If so you should check the shape of the masks you provide. Below, we provide a working example with masks, so you can see the shape that it should be.

We hope this helps and if you still have issues, don't hesitate to tell us, we will do our best to help !

While investigating your issue, we also saw a few improvements that could be made in the errors messages that are raised so we updated the main branch with those.


```python

# Example with 2D data and masks
import torch

from multivae.data.datasets import IncompleteDataset
from multivae.models import MVAE, MVAEConfig

dataset = IncompleteDataset(
    data={'mod1': torch.randn(100, 5),
          'mod2': torch.randn(100, 10)},
    masks={'mod1': torch.bernoulli(torch.ones(100,)*0.9), # random masks to simulate missing data.(90% of first modality is present)
           # Since we have 100 samples, we have 100 values in each modality mask. For each modality, the mask's shape is (100,).
           'mod2': torch.ones(100,)}, # all data is present in this modality
)

model_config = MVAEConfig(
    n_modalities = 2,
    input_dims= {'mod1': (5,), 
                'mod2': (10,)
                },
    latent_dim= 8,
)

model = MVAE(model_config)

# Define a trainer

from multivae.trainers import BaseTrainer, BaseTrainerConfig

training_config = BaseTrainerConfig(
    learning_rate=1e-3, 
    num_epochs=1
)

trainer = BaseTrainer(model , dataset, training_config=training_config)

trainer.train()
```
